.coderabbit.yaml

```
@@ -0,0 +1,82 @@
+
+language: en-US
+early_access: false
+tone_instructions: "Use a professional, institutional-grade tone; prioritize security, performance"
+
+reviews:
+  profile: chill
+  request_changes_workflow: false
+  high_level_summary: true
+  high_level_summary_placeholder: |
+    ## CodeRabbit Summary
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)